make/addons.mk

```diff
@@ -59,3 +66,13 @@ update-addon.kube-vip: ; $(info $(M) updating kube-vip manifests)
 generate-helm-configmap:
 	go run hack/tools/helm-cm/main.go -kustomize-directory="./hack/addons/kustomize" -output-file="./charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml"
 	./hack/addons/add-warning-helm-configmap.sh
+
+# Set only the supported CSI providers for each provider.
+.PHONY: configure-csi-providers.%
```

Comment:
nice - can we move all the scripting to a file and call it like the rest of this file does? it helps keep things neat

Reply:
Done!